src/builders/base_builders.py

```diff
@@ -109,11 +105,16 @@ def _get_file_paths(self):
         """Get all rel_path keys from the entity dict.
 
         >>> files = [{ "rel_path": "path/to/file" }, { "rel_path": "path/to/other_file" }]
-        >>> vc = ViewConfBuilder(
+        >>> vc = _ConcreteBuilder(
         ...   entity={"uuid": "uuid", "files": files},
         ...   groups_token='groups_token',
         ...   assets_endpoint='https://example.com')
         >>> vc._get_file_paths()
         ['path/to/file', 'path/to/other_file']
         """
         return [file["rel_path"] for file in self._entity["files"]]
+
+
+class _ConcreteBuilder(ViewConfBuilder):  # pragma: no cover
+    def get_conf_cells(self):
+        pass
```

Comment:
Maybe a little rusty in my OOP from living in the react world, but why the abstract class when the subclass only overrides the abstract method with `pass`?

Reply:
It was unclear that this class was created solely for testing and that there are other, more meaningful subclasses of `ViewConfBuilder`.